playwright/e2e/login/login.spec.ts

```diff
@@ -31,12 +31,18 @@ test.describe("Login", () => {
             await homeserver.registerUser(username, password);
         });
 
-        test("logs in with an existing account and lands on the home screen", async ({
+        test("Loads the welcome page by default; then logs in with an existing account and lands on the home screen", async ({
             page,
             homeserver,
             checkA11y,
         }) => {
-            await page.goto("/#/login");
```

Comment:
A direct link to `/#/login` is well-tested by myriad other tests (including the one just below). I don't think we're losing anything by making this test bounce via the welcome page.